.prettierignore

```diff
@@ -88,3 +88,4 @@ apps/site/site-map.json
 apps/site/test-results
 packages/block-template-html/dev/src
 packages/mock-block-dock/src/version.ts
+packages/@blockprotocol/type-system/wasm
```

Comment:
I'm assuming you're referring to 
`packages/@blockprotocol/type-system/.rollup.cache`
being missing here.

I've added it.

Reply:
Can we sync all `*ignore` files in root by copy-pasting the whole section? `.gitignore`, `.markdownlintignore`, `.prettierignore`. This will make further syncing less confusing.

Reply:
Ignore files don't support includes, so we have to sync their contents manually. I wish there was a linting tool for this.

Reply:
I had missed the markdownignore in my search, apologies.